standard.tex

```diff
@@ -66,15 +69,41 @@ \subsection{General Wiring \& Assembly}
 \end{itemize}
+%end of wiring and assembly subsection -- }}}
 
+%start of software subsection -- {{{
 \subsection{Software}
 \begin{itemize}
 \item If your system has any software anywhere, that software should be unit tested.
 \item If your system requires a computer program to be used, that program counts as mission critical. It should run on at least two machines and at least two people should be trained to use it.
 \item All mission critical software should be expressible (and accompanied by) less than 100 lines of pseudocode.
 \item All software should use a VCS (probably git), and all final versions of your software should go with your development logs (see Documentation Standards) on the drive.
 \end{itemize}
+%end of software subsection -- }}}
 
+%start of pcb subsection -- {{{
+\subsection{PCB design}
+\begin{itemize}
+\item Watch this EEVblog video: \url{https://www.youtube.com/watch?v=JrH_itjMDjo}. It's like an hour and a half long, but well worth watching.
+\item Given the choice, please draft your schematics and board layouts in KiCad. It's free (GPLv3), used by the rest of the team, and runs on all platforms. \url{http://kicad-pcb.org/}
+\item if your PCBs are meant to be screwed onto something, put those mounting holes on your layout, and if at all possible, use M3 screws (for standardization purposes, if you have a space restriction, smaller screws are fine).
+\item Before ordering any components, make a list of all components in your design, and go through their data sheets one by one and ensure that they meet or exceed the requirements of your design. Do not just trust the Digikey table values, they are occasionally mislabeled.
+\item The cheapest board that can be purchased in small quantities is a 2 sided 1oz 5cm by 10cm 10/10 (10 mil (.254mm) min clearance and min trace width) 0.4mm smallest drill size board. If at all possible, try to make your design fit that requirement.
+\item If a pin is listed as NC on a data sheet, don't pull it to ground. Just leave it unconnected. Unused IO pins are a different story, they may need to be shorted to ground or VCC. Consult your data sheet in this instance.
+
+\item If you plan to hand-solder your boards for assembly:
+\begin{itemize}
+\item Watch this video \url{https://www.youtube.com/watch?v=J5Sb21qbpEQ} (EEVblog soldering tutorial). Then watch part 2. Then watch part 3. Yes that involves an hour and a half time commitment, but after watching those 3 you'll know exactly as much as I do about soldering.
+\item Use pads that are longer than you need. KiCad has specific footprints whose names end in \_handsoldering, which have extended pads for easier hand-soldering.
+\item Use flux. It makes surface mount soldering orders of magnitude cheaper. 10ml syringes are less than \$20 at Sayal, and should last for a while.
+\item SOT-23 and 0805 SMT components are quite easily hand-solderable. SOT-23-5 are also doable, but are harder. I've never attempted anything smaller than those.
```

Comment:
Last sentence doesn't add much.

Reply:
It's there to say that "maybe it's possible to solder 0603 stuff by hand, but I don't know"

Reply:
Maybe rephrase? There isn't really an "I" in this doc anymore, especially if more people add things.